plugins/hosts/linux/cap/nfs.rb

```diff
@@ -261,6 +261,19 @@ def self.nfs_running?(check_command)
           Vagrant::Util::Subprocess.execute(*Shellwords.split(check_command)).exit_code == 0
         end
 
+        def self.modinfo_path
+          path = Vagrant::Util::Which.which("modinfo")
+          return path if path
+
+          folders = ["/sbin"]
+          folders.each do |folder|
+            path = "#{folder}/modinfo"
+            return path if File.file?(path)
+          end
+
+          "modinfo"
+        end
```

Comment:
We have a pattern for methods like this. We can cache the value locally to prevent us from looking up this value multiple times during a Vagrant run:

```ruby
def self.modinfo_path
  if !defined?(@_modinfo_path)
    @_modinfo_path = Vagrant::Util::Which.which("modinfo")

    if @_modinfo_path.to_s.empty?
      folders = ["/sbin"]
      folders.each do |folder|
        path = "#{folder}/modinfo"
        if File.file?(path)
          @_modinfo_path = path
          break
        end
      end
    end

    if @_modinfo_path.to_s.empty?
      @_modinfo_path = "modinfo"
    end
  end
  @_modinfo_path
end
```

I haven't tested that, but something like this! :sweat_smile: :pray:

Reply:
Thanks! 🙏 You were right on.  I updated the method and added a test for when the value is cached.